lib/renderer/api/web-frame.ts

```diff
@@ -51,38 +49,42 @@ class WebFrame extends EventEmitter {
 
   // Deprecations
   // TODO(nitsakh): Remove in 6.0
-  setIsolatedWorldSecurityOrigin (worldId, securityOrigin) {
+  setIsolatedWorldSecurityOrigin (worldId: number, securityOrigin: string) {
     deprecate.warn('webFrame.setIsolatedWorldSecurityOrigin', 'webFrame.setIsolatedWorldInfo')
     binding.setIsolatedWorldInfo(this.context, worldId, { securityOrigin })
   }
 
-  setIsolatedWorldContentSecurityPolicy (worldId, csp) {
+  setIsolatedWorldContentSecurityPolicy (worldId: number, csp: string) {
     deprecate.warn('webFrame.setIsolatedWorldContentSecurityPolicy', 'webFrame.setIsolatedWorldInfo')
     binding.setIsolatedWorldInfo(this.context, worldId, {
       securityOrigin: window.location.origin,
       csp
     })
   }
 
-  setIsolatedWorldHumanReadableName (worldId, name) {
+  setIsolatedWorldHumanReadableName (worldId: number, name: string) {
     deprecate.warn('webFrame.setIsolatedWorldHumanReadableName', 'webFrame.setIsolatedWorldInfo')
     binding.setIsolatedWorldInfo(this.context, worldId, { name })
   }
 }
 
 // Populate the methods.
 for (const name in binding) {
-  if (!name.startsWith('_')) { // some methods are manully populated above
-    WebFrame.prototype[name] = function (...args) {
+  if (!name.startsWith('_')) { // some methods are manually populated above
+    // TODO(felixrieseberg): Once we can type web_frame natives, we could
+    // use a neat `keyof` here
+    (WebFrame as any).prototype[name] = function (...args: Array<any>) {
       return binding[name](this.context, ...args)
     }
   }
 }
 
 // Helper to return WebFrame or null depending on context.
 // TODO(zcbenz): Consider returning same WebFrame for the same frame.
-function getWebFrame (context) {
+function getWebFrame (context: Window) {
   return context ? new WebFrame(context) : null
 }
 
-module.exports = new WebFrame(window)
+const _webFrame = new WebFrame(window)
```

Comment:
We can't `export default new WebFrame(window)` here?

Reply:
We can, but this way it's named if anything goes wrong!